code/modules/trade/datums/trade_stations_presets/nt_cruisers.dm

```diff
@@ -0,0 +1,47 @@
+/datum/trade_station/nt_cruisers
+	icon_states = "nt_cruiser"
+	spawn_cost = 1
+	markup = 1
```

Comment:
I'd put a bit of a markup on this so the NT on the ship can kind of keep their monopoly on pouches.